app/models/user.rb

```diff
@@ -54,6 +54,10 @@ def staff?
     site_admin
   end
 
+  def owns_assignments?(organization)
```

Comment:
The naming here doesn't really fit the method, could we update this to something like `owns_all_assignments_for?(organization)` ?